packages/storycap/src/client/trigger-screenshot.ts

```diff
@@ -75,8 +75,15 @@ function consumeOptions(win: StorycapWindow, storyKey: string): ScreenshotOption
   return result;
 }
 
-function stock(opt: ScreenshotOptions = {}) {
-  const storyKey = url2StoryKey(location.href);
+function stock(opt: ScreenshotOptions = {}, context: any) {
+  let storyKey: string | undefined = undefined;
+
+  if (!!context.kind && !!context.story) {
```

Comment:
Might be the `context` undefined ? 

```ts
if (context && !!context.kind && !!context.story) {
```

Reply:
Ok. I have added this extra check. Does the e2e test work now?